test/tools/ossfuzz/abiV2ProtoFuzzer.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <fstream>
+#include <src/libfuzzer/libfuzzer_macro.h>
+#include <test/tools/ossfuzz/protoToAbiV2.h>
+#include <test/tools/evmoneRunner.h>
+#include "abiV2FuzzerCommon.h"
+#include <src/text_format.h>
+
+
+using namespace dev::test::abiv2fuzzer;
+using namespace external::evmone;
+using namespace std;
+using namespace dev;
+
+#if 1
+static uint64_t totalFailedTests = 0;
+#endif
+
+DEFINE_PROTO_FUZZER(Contract const& _input)
+{
+	ProtoConverter converter;
+	string contract_source = converter.contractToString(_input);
```

Comment:
`string contract_source = ProtoConverter{}.contractToString(_input);`

 - no need to keep it alive for the whole function